service/controller/resource/generic/create.go

```diff
@@ -46,8 +46,6 @@ func updateMeta(c, d metav1.Object) {
 	d.SetCreationTimestamp(c.GetCreationTimestamp())
 	d.SetDeletionTimestamp(c.GetDeletionTimestamp())
 	d.SetDeletionGracePeriodSeconds(c.GetDeletionGracePeriodSeconds())
-	d.SetLabels(c.GetLabels())
-	d.SetAnnotations(c.GetAnnotations())
```

Comment:
Was this intended?

Reply:
I'm on it as the namespace change is making some of it not work anymore :D
Yes it is, otherwise, we will never be able to update labels and annotations :( 
I am thinking we can reenable it in a subsequent release